diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter2/comments/CommentedCodeTest.kt

```diff
@@ -256,4 +256,42 @@ class CommentedCodeTest : LintTestBase(::CommentsRule) {
             """.trimMargin(),
             LintError(1, 1, ruleId, "${COMMENTED_OUT_CODE.warnText()} public fun someFunc(name: String): Boolean =", false))
     }
+
+
+    @Test
+    @Tag(WarningNames.COMMENTED_OUT_CODE)
+    fun `should not trigger on multiline comments #1`() {
+        lintMethod(
+                """
+            |/*
+            |
+            |   Copyright 2018-2020 Charles Korn.
```

Comment:
```suggestion
            |   Copyright 2018-2020 John Doe.
```
Let's not use real people's names here :)

Reply:
Fixed